docs/content/https/acme.md

```diff
@@ -25,7 +25,9 @@ You can read more about this retrieval mechanism in the following section: [ACME
 
 !!! warning "Defining an [ACME challenge type](#the-different-acme-challenges) is a requirement for a certificate resolver to be functional."
 
-!!! important "Defining a certificates resolver does not result in all routers automatically using it. Each router that is supposed to use the resolver must [reference](../routing/routers/index.md#certresolver) it."
+!!! important "Defining a certificate resolver does not result in all routers automatically using it. Each router that is supposed to use the resolver must [reference](../routing/routers/index.md#certresolver) it."
+
+!!! important "Certificate resolvers are defined with the `certificatesResolvers` key. Note that the key uses the plural form of `certificates`."
```

Comment:
```suggestion
```

Reply:
@mpl Could you give me some feedback about whether this or a similar change would be welcomed in a different PR? Maybe I'm the only person not copy-pasting their config from the docs, but I do think this could be friendlier to people new to Traefik.